tests/Opus/DocumentFinder/DefaultDocumentFinderTest.php

```diff
@@ -0,0 +1,764 @@
+        $this->assertFalse(in_array($invisibleFileDocId, $foundIds), 'Expected no id of Document with invisible file in OAI');
+    }
+
+    public function testSetEmbargoDateBefore()
+    {
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-16');
+        $doc1Id = $doc->store();
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-14');
+        $doc2Id = $doc->store();
+
+        $docfinder = new DefaultDocumentFinder();
+        $docfinder->setEmbargoDateBefore('2016-10-15');
+        $foundIds = $docfinder->getIds();
+
+        $this->assertCount(1, $foundIds);
+        $this->assertContains($doc2Id, $foundIds);
+        $this->assertNotContains($doc1Id, $foundIds);
+    }
+
+    public function testSetEmbargoDateAfter()
+    {
+        $this->markTestSkipped('TODO DOCTRINE DBAL Issue #129: Function setEmbargoDateAfter() is no part of the DocumentFinderInterface');
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-16');
+        $doc1Id = $doc->store();
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-14');
+        $doc2Id = $doc->store();
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-15');
+        $doc3Id = $doc->store();
+
+        $docfinder = new DefaultDocumentFinder();
+        $docfinder->setEmbargoDateAfter('2016-10-15');
+        $foundIds = $docfinder->getIds();
+
+        $this->assertCount(2, $foundIds);
+        $this->assertContains($doc1Id, $foundIds);
+        $this->assertContains($doc3Id, $foundIds);
+        $this->assertNotContains($doc2Id, $foundIds);
+    }
+
+    public function testSetEmbargoDateRange()
+    {
+        $this->markTestSkipped('TODO DOCTRINE DBAL Issue #129: Function setEmbargoDateRange() is no part of the DocumentFinderInterface');
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-16'); // not in range
+        $doc1Id = $doc->store();
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-13'); // not in range
+        $doc2Id = $doc->store();
+
+        $doc = new Document();
+        $doc->setEmbargoDate('2016-10-14'); // in range
+        $doc3Id = $doc->store();
+
+        $docfinder = new DefaultDocumentFinder();
+        $docfinder->setEmbargoDateRange('2016-10-14', '2016-10-16');
+        $foundIds = $docfinder->getIds();
+
+        $this->assertCount(1, $foundIds);
+        $this->assertContains($doc3Id, $foundIds);
+        $this->assertNotContains($doc1Id, $foundIds);
+        $this->assertNotContains($doc2Id, $foundIds);
+    }
+
+    /**
+     * Tests from a perspective of two days in the future to avoid the need to manipulate ServerDateModified.
+     */
+    public function testFindDocumentsWithExpiredEmbargoDateForUpdatingServerDateModified()
+    {
+        $this->markTestSkipped('TODO DOCTRINE DBAL Issue #129: Function setEmbargoDateBeforeNotModifiedAfter() is no part of the DocumentFinderInterface');
```

Comment:
Die Funktion wurde aufgeteilt bzw. jetzt müssen `setEmbargoDateBefore` und `setNotModifiedAfterEmbargoDate` verwendet werden. Der Test sollte dahingehend umgebaut werden, so dass die Idee, die Funktionalität weiterhin getestet wird.

Reply:
In den betroffen Tests habe ich jetzt die neuen Funktionen verwendet.